skutil/preprocessing/tests/test_impute.py

```diff
@@ -144,6 +144,7 @@ def test_selective_imputer():
     assert_fails(SelectiveImputer(def_fill=['a', 'b', 'c']).fit, TypeError, a)
     assert_fails(SelectiveImputer(def_fill='a').fit, TypeError, a)
     assert_fails(SelectiveImputer(def_fill=[1, 2, 'a']).fit, TypeError, a)
+    assert_fails(SelectiveImputer().fit, TypeError, a)
```

Comment:
This is where CI is failing. Why do we expect it to fail here? (I've been trying to put a little comment out to the side, like "this will fail because __x__") Do you recall?